fund_america.gemspec

```diff
@@ -28,15 +28,16 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   # Development Dependencies
-  spec.add_development_dependency "bundler", "~> 1.9"
+  spec.add_development_dependency "bundler", "~> 2.0.1"
```

Comment:
I see we use bundler `1.17.3` in BF. It would be good if dev dependencies of this gem would be compatible with these used in BF